src/components/FormLogin.vue

```diff
@@ -21,92 +21,208 @@ import { defineComponent, ref, reactive } from 'vue';
+      email: '',
+    });
+
+    const isPassword = ref(true);
+    const formState = ref<'login' | 'password-reset'>('login');
 
     const isValid = (val: string): boolean => val?.length > 0;
 
-    const onSubmit = () => {
-      emit('formSubmit');
+    const onSubmitLogin = () => {
+      // noop
+    };
+
+    const onSubmitPasswordReset = () => {
       // noop
     };
 
     return {
       formLogin,
+      formPasswordReset,
+      formState,
       isPassword,
       isValid,
-      onSubmit,
+      onSubmitLogin,
+      onSubmitPasswordReset,
     };
   },
 });
 </script>
 
 <template>
-  <q-form @submit.prevent="onSubmit">
-    <!-- Input: email -->
-    <div data-cy="form-login-email">
-      <!-- Label -->
-      <label for="form-login-email" class="text-caption text-bold">
-        {{ $t('login.form.email') }}
-      </label>
-      <!-- Input -->
-      <q-input
-        dense
-        outlined
-        v-model="formLogin.email"
-        :rules="[(val) => isValid(val) || $t('login.form.emailReqired')]"
-        bg-color="grey-1"
-        id="form-login-email"
-        name="subject"
-        class="q-mt-sm"
-        data-cy="form-login-email-input" />
-    </div>
-    <!-- Input: password -->
-    <div data-cy="form-login-password">
-      <!-- Label -->
-      <label for="form-login-password" class="text-caption text-bold">
-        {{ $t('login.form.password') }}
-      </label>
-      <!-- Input -->
-      <q-input
-        dense
-        outlined
-        hide-bottom-space
-        bg-color="grey-1"
-        v-model="formLogin.password"
-        id="form-login-password"
-        :type="isPassword ? 'password' : 'text'"
-        :rules="[(val) => isValid(val) || $t('login.form.passwordRequired')]"
-        class="q-mt-sm"
-        data-cy="form-login-password-input"
+  <div v-if="formState === 'login'" class="text-grey-10" data-cy="form-login">
+    <div class="q-my-lg">
+      <h1
+        class="text-h5 text-bold grey-10 q-my-none"
+        data-cy="form-title-login"
       >
-        <!-- Icon: show password -->
-        <template v-slot:append>
-          <q-icon color="primary" :name="isPassword ? 'visibility_off' : 'visibility'" class="cursor-pointer" size="18px" @click="isPassword = !isPassword" data-cy="form-login-password-icon" />
-        </template>
-      </q-input>
-      <!-- Link: fogotten password -->
-      <div class="flex justify-end q-mt-sm">
-        <a href="#" class="text-primary text-caption" data-cy="form-login-forgotten-password">{{ $t('login.form.forgottenPassword') }}</a>
+        {{ $t('login.form.titleLogin') }}
+      </h1>
+    </div>
+    <!-- Form: login -->
+    <q-form @submit.prevent="onSubmitLogin">
+      <!-- Input: email -->
+      <div data-cy="form-login-email">
+        <!-- Label -->
+        <label for="form-login-email" class="text-caption text-bold">
+          {{ $t('login.form.labelEmail') }}
+        </label>
+        <!-- Input -->
+        <q-input
```

Comment:
Missing `type` prop `type="email"`.

Reply:
Email validation will be handled by the custom validation rules, not the `type` property.